Sources/ClientRuntime/Networking/Streaming/ByteStream.swift

```diff
@@ -7,45 +7,12 @@
 import AwsCommonRuntimeKit
 import class Foundation.FileHandle
 
-/// A stream of bytes.
 public enum ByteStream {
-    /// A stream of bytes represented as a `Data` object.
     case data(Data?)
-
-    /// A stream of bytes represented as a `Stream` object.
-    /// - Note: This representation is recommended for large streams of bytes.
     case stream(Stream)
-}
-
-extension ByteStream {
-    /// Returns ByteStream from a Data object.
-    /// - Parameter data: Data object to be converted to ByteStream.
-    /// - Returns: ByteStream representation of the Data object.
-    public static func from(data: Data) -> ByteStream {
-        return .data(data)
-    }
-
-    /// Returns ByteStream from a FileHandle object.
-    /// - Parameter fileHandle: FileHandle object to be converted to ByteStream.
-    /// - Returns: ByteStream representation of the FileHandle object.
-    public static func from(fileHandle: FileHandle) -> ByteStream {
-        return .stream(FileStream(fileHandle: fileHandle))
-    }
+    case none
```

Comment:
Naming a case `.none` can lead to confusion because a Swift optional is actually an `enum` with a case also named `.none`:

```
enum Optional<T> {
    case some(T)
    case none
}
```

Consider another name such as `noStream` to prevent hard to find bugs down the line.